.github/workflows/releases.yml

```
@@ -19,7 +19,7 @@ jobs:
       - uses: actions/setup-python@v4.7.1
         name: Install Python
         with:
-          python-version: '3.8'
+          python-version: '3.9'
```

I lean toward publishing with the latest python release that we support.
```suggestion
          python-version: '3.11'
```